util/config.py

```diff
@@ -56,11 +56,6 @@ def initialize_globals():
     if not FLAGS.summary_dir:
         FLAGS.summary_dir = xdg.save_data_path(os.path.join('deepspeech', 'summaries'))
 
-    # Standard session configuration that'll be used for all new sessions.
-    c.session_config = tf.ConfigProto(allow_soft_placement=True, log_device_placement=FLAGS.log_placement,
```

Comment:
I remember some people explicitely opened an issue to have those flags, are we okay removing them ?

Reply:
This API has been deprecated by TensorFlow, so we would lose it anyway in a future update. I removed it to silence the warning since we've never touched these flags. Your call whether to keep this change or not.

Reply:
I'm more concerned about the inter / intra parallelism flags, do they also go away with no alternative ?

Reply:
AFAICT all of `ConfigProto` was deprecated.

Reply:
(TF 2.0 doesn't have sessions)